src/content/en/2021/cdn.md

```diff
@@ -179,6 +184,8 @@ We also see providers like:
 * Google,
 * AWS
 
+{# TODO: these lists feel disruptive when reading. Maybe they can be combined into a sentence or 2? #}
```

Comment:
@Navaneeth-akam

Reply:
Yes. This one had mixed opinion from the reviewers. They felt having it as bullet points is better. But i do feel putting them in a sentence makes more sense. We can go ahead and put them in a sentence